tests/test_markitdown.py

```diff
@@ -87,6 +87,13 @@
     "data:image/svg+xml,%3Csvg%20width%3D",
 ]
 
+CSV_CP932_TEST_STRINGS = [
+    "名前,年齢,住所",
+    "佐藤太郎,30,東京",
+    "三木英子,25,大阪",
+    "髙橋淳,35,名古屋",
+]
+
```

Comment:
This started appearing weird for some reason on GitHub diff:

<img width="1083" alt="image" src="https://github.com/user-attachments/assets/98e94c3b-fe7e-4e81-9c46-488dfec2b9f0" />

But if you open the file it shows it correctly:

<img width="489" alt="image" src="https://github.com/user-attachments/assets/214338ac-3c23-4537-a5fe-eeceeb140f3d" />

<img width="830" alt="image" src="https://github.com/user-attachments/assets/246d57a2-a37d-4c5f-af29-210e00b57498" />